app/components/post_list/post/avatar/index.tsx

```diff
@@ -176,4 +176,4 @@ const withPost = withObservables(['post'], ({database, post}: {post: PostModel}
     };
 });
 
-export default withDatabase(withPost(React.memo(Avatar)));
+export default withDatabase(withPost(Avatar));
```

Comment:
I piggybacked to this PR the removal of several React.memo that may cause problems in the future. There are still some in the code, but seemed that may not give so many problems.

Reply:
I guess the problem will keep the same, with the issue moving to the whole database variable used in "withPost". It might be solved with `React.memo(withDatabase(withPost(Avatar)))` but I think it is better to let React and watermelonDB to do their thing, and optimize later if we see any real issue.